src/translate_c.zig

```diff
@@ -3949,17 +3984,6 @@ fn transFloatingLiteral(c: *Context, expr: *const clang.FloatingLiteral, used: R
             .{format},
         ),
     }
-    // TODO use something more accurate
-    var dbl = expr.getValueAsApproximateDouble();
```

Comment:
I removed this comment as from what I can see, despite having "approximate" in its name, this function is accurate for 64-bit and smaller IEEE floats, but looking at it now, maybe it was referring to using the source location to retain the exact representation of the float (like with float literals in macros)?